packages/headless/src/controllers/breadcrumb-manager/headless-breadcrumb-manager.ts

```diff
@@ -108,7 +108,7 @@ export const buildBreadcrumbManager = (
     return Object.keys(engine.state.categoryFacetSet)
       .map((facetId) => {
         return {
-          field: engine.state.categoryFacetSet[facetId].field,
+          field: engine.state.categoryFacetSet[facetId]?.request.field || '',
```

Comment:
Could we not do `...[facetId]!.request` here since we know its going to be defined because we're iterating through the keys of the facet set?

Reply:
Yes, indeed!